### PR TITLE
fix: remove setup() and move code to data() in Results.vue

### PR DESCRIPTION
**Pull request by Chartman123**
Signed-off-by: Christian Hartmann <chris-hartmann@gmx.de>

### CODE REVIEW

File: src/views/Results.vue [hunk 1]
```diff
@@ -305,8 +305,18 @@ export default {
...
+
+			isDownloadActionOpened: false,
+			loadingResults: true,
+
+			picker: null,
+			showSummary: true,
+			showConfirmDeleteDialog: false,
+			showLinkedFileNotAvailableDialog: false,
+
 			isMobile: useIsSmallMobile(),
```
Comment (susnux) [inline src/views/Results.vue:320]: This **must** be in `setup`. (even if this specific case might work, composables should always and only be used in `setup` context).

**Comment by Chartman123** [comment]
Ok, could you please fix my code then?

**Comment by susnux** [comment]
> @[susnux](https://github.com/susnux) susnux merged commit [56f410a](https://github.com/nextcloud/forms/commit/56f410a34be953c7ea09e371bda34d460509cb97) into main [Apr 15, 2024](https://github.com/nextcloud/forms/pull/2065#event-12471846466)

I am pretty sure I merged the other PR not this one.

**Comment by Chartman123** [comment]
Ok, should we revert it partially? I just read that setup() is only supported with vue3, that's why I moved everything into data().

**Comment by susnux** [comment]
> Ok, should we revert it partially? I just read that setup() is only supported with vue3

No, `setup` is supported since Vue 2.7
Everything in `data` will be made reactive, things in `setup` not. In setup you use the Composition API to make things reactive (e.g. `const foo = ref(4)`).

But some things like composables need to be defined in `setup` to allow them to register all life-cycle hooks like `mounted`.

**Pull request merged**